### DISCUSSION TITLE
In-app update feature in nativescript

### DISCUSSION THREAD

**Discussion by chinnawip** [community/community#68222] [Programming Help]
### Body

Hi,
I need a help regarding the in-app update feature that how to implement it in nativescript android app. Would be great if anyone knows any plugin/links available for this and share it.
Thanks and Regards
Chinna

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ebubekirgungor** [comment]
You can do it with AppSync plugin. Here is a blog post explaining how to do this: [https://blog.nativescript.org/enable-live-updates-of-your-ios-and-android-apps-using-nativescript-appsync/](https://blog.nativescript.org/enable-live-updates-of-your-ios-and-android-apps-using-nativescript-appsync/)
Github page of the plugin: [https://github.com/NativeScript/nativescript-app-sync](https://github.com/NativeScript/nativescript-app-sync)

**Reply by chinnawip** [reply]
Thanks **ebubekirgungor** for your reply. However the solution you have given is not suitable for our need. So I made it working using nativecode as given in this link **https://github.com/chronogolf/nativescript-store-update/issues/30**

**Reply by im-shery** [reply]
Hi, did you make some tweaks to make it work as I always get API_NOT_AVAILABLE when I am testing it.

**Reply by chinnawip** [reply]
Hi, sorry for the late reply. Actually I followed this link https://github.com/chronogolf/nativescript-store-update/issues/30#issuecomment-622454891
